data/css/modules/layout/_paginatedScrolling.scss

```diff
@@ -0,0 +1,14 @@
+scrollbar.vertical slider {
+    min-height: 100px;
+}
+
+.LayoutPaginatedScrolling {
+    background-color: $background-light-color;
+    .text-button {
+      background-color: $primary-medium-color;
+      color: white;
+      padding: 5px;
+      margin: 5px;
+      border-radius: 10px;
+    }
+}
```

Comment:
OK I just gave it the exact same styling as the TabButton, except without the `go-down-symbolic` because in this case you're not going down anywhere, just loading more content. If we get more designs for this button in the future I can open another issue to add those.